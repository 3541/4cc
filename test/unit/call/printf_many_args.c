#include <stdio.h>

int main(void) {
    printf("This %s string %s many arguments. %d %d %d %d %d.\n", "format", "requires", 4, 5, 6, 7,
           8);
}
