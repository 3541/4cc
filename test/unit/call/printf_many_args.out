This format string requires many arguments. 4 5 6 7 8.
